src/docs/asciidoc/data-access.adoc

```diff
@@ -2730,17 +2730,15 @@ The following query finds and populates a single domain object:
 [source,java,indent=0,subs="verbatim,quotes",role="primary"]
 .Java
 ----
+	RowMapper<Actor> actorRowMapper = (resultRest, rowNum) -> {
```

Comment:
`resultRest` --> `resultSet`

Please make that change consistently in this PR.